Documentation/getting-started-tizen.md

```diff
@@ -16,26 +16,26 @@ title: Tizen GBS
 ### Build without options
 
 ```bash
-$ gbs build
+$ gbs -c .TAOS-CI/.gbs.conf build
```

Comment:
Does the name of config file have to be `.gbs.conf`? If it doesn't, I want to suggest another name like `.gbs-taos.conf`.

Reply:
> Does the name of config file have to be .gbs.conf? 

@helloahn  , Thank you commenting on the unique file name. I want to keep the default file name (e.g., .gbs.conf) that is explained at the official gbs guide because the folder name will be unique. If the same file name will be confused to TAOS-CI users in the future, let's consider unique file name (e.g., .gbs-taos.conf) as well as unique folder name (./.TAOS-CI/) again. 😺 
* https://docs.tizen.org/platform/reference/gbs/gbs.conf/

Reply:
Thank you for your explanation :)